src/components/Unknown/BuildScreen/Builder/CPUBuilder/index.tsx

```diff
@@ -68,8 +52,17 @@ const CPUBuilder: React.FC = () => {
       selectedId={selectedId}
       expand={expand}
       toggleAccordion={toggleAccordion}
+      selectedProduct={selectedProduct}
     >
-      {products}
+      {CPUs &&
```

Comment:
if CPUs is loading show <CirclePreloader />

Reply:
agree